dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-7.0.feature

```diff
@@ -0,0 +1,28 @@
+-include= ~../cnf/resources/bnd/feature.props
```

Comment:
Does this file have the right name? It says appserver.javaee-7.0.feature

Reply:
you're right -- need to fix that